src/Aquifer.Public.API/Endpoints/Resources/Collections/Get/Endpoint.cs

```diff
@@ -0,0 +1,211 @@
+        }
+        if (invalidLanguageCodes is not [])
+        {
+            ThrowError(r => r.LanguageCodes, $"Invalid 'Language Codes': \"{string.Join("\", \"", invalidLanguageCodes)}\"");
+        }
+
+        var parentResource = await GetParentResourceAsync(_dbContext.Database.GetDbConnection(), req.Code, ct);
+        if (parentResource is null)
+        {
+            await SendNotFoundAsync(ct);
+            return;
+        }
+
+        var localizations = await GetParentResourceLocalizationWithResourceContentCountAsync(
+            _dbContext.Database.GetDbConnection(),
+            parentResource.Id,
+            validLanguageIds,
+            ct);
+
+        var languageCodeByIdMap = await _cachingLanguageService.GetLanguageCodeByIdMapAsync(ct);
+
+        var response = new Response
+        {
+            Code = parentResource.Code,
+            DisplayName = parentResource.DisplayName,
+            ShortName = parentResource.ShortName,
+            ResourceType = parentResource.ResourceType,
+            LicenseInfo = parentResource.LicenseInfo,
+            AvailableLanguages = localizations
+                .Select(l => new AvailableLanguageResponse
+                {
+                    LanguageId = l.LanguageId,
+                    LanguageCode = languageCodeByIdMap[l.LanguageId],
+                    DisplayName = l.DisplayName,
+                    ResourceItemCount = l.ResourceCount,
+                })
+                .ToList(),
+        };
+
+        await SendOkAsync(response, ct);
+    }
+
+    private static async Task<ParentResource?> GetParentResourceAsync(DbConnection dbConnection, string code, CancellationToken ct)
+    {
+        const string query = """
+            SELECT
+                pr.Id,
+                pr.Code,
+                pr.DisplayName,
+                pr.ShortName,
+                pr.ResourceType,
+                pr.LicenseInfo
+            FROM
+                ParentResources pr
+            WHERE
+                pr.Code = @code AND
+                pr.[Enabled] = 1;
+            """;
+
+        return await dbConnection.QuerySingleOrDefaultAsync<ParentResource>(new CommandDefinition(
+            query,
+            new
+            {
+                code
+            },
+            cancellationToken: ct));
+    }
+
+    private sealed record ParentResource(
+        int Id,
+        string Code,
+        string DisplayName,
+        string ShortName,
+        ResourceType ResourceType,
+        string LicenseInfo);
+
+    private static async Task<IReadOnlyList<ParentResourceLocalizationWithResourceContentCount>>
+        GetParentResourceLocalizationWithResourceContentCountAsync(
+            DbConnection dbConnection,
+            int parentResourceId,
+            IReadOnlyList<int>? languageIds,
+            CancellationToken ct)
+    {
+        var parameters = new DynamicParameters(new { parentResourceId });
```

Comment:
`DynamicParameters` are a nice Dapper feature that allows starting with static parameters and then dynamically adding in more in conditional statements without having to deal with ADO.NET pain.